src/legacy/scss/partials/_shame.scss

```diff
@@ -43,6 +43,9 @@
         z-index: 1001;
         width: 500px;
         height: $lineheight * 17;
+        &--height-50{
+            height: 400px;
```

Comment:
Could we possibly use the `$lineheight` variable instead hardcoding the height, to keep things as consistent as possible? For example
```
height: $lineheight * 21;
```